.changeset/yellow-suns-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/auth': patch
+---
+
+Prevent React Native logging a warning about deprecation of `AsyncStorage` if user has provided the non-deprecated version.
```

Comment:
The preposition could clarify a little: "Prevent React Native from logging..."

Reply:
Done.